hw03.py

```diff
@@ -0,0 +1,41 @@
+x = input("Write something:")
+for a in x:
+        if x.isdigit():
+            if int(x) % 2 == 0:
+                print("This is a number")
+                print("Number is even")
+
+            else:
+                print("This is a number")
+                print("Number is odd")
+
+        elif x.isalpha():
+            if x.islower():
+                print("This is a letter")
+                print("This is uppercase letter")
```

Comment:
The comment says that the letter s uppercase, but this code run for lowercase letters. Looks like you messed it up